mod/discussions/start.php

```diff
@@ -11,6 +11,12 @@
 function discussion_init() {
 
 	elgg_register_page_handler('discussion', 'discussion_page_handler');
+	
+	elgg_register_menu_item('site', [
```

Comment:
I guess I'm begging if this must be registered, make it a setting, and it would depend on "groups" being active. But it's already so easy to add a site menu item.